### PR TITLE
Ft reset ps 31

### PR DESCRIPTION
**Pull request by tysepa**
This is a reset page 

Here you can reset your password even if you have forgotten 
and when you enter your email you have to create your new password 

How should this be manually tested?
Clone the repo

checkout to branch ft-resetPs-31

Run npm install to install modules

Run npm run start to start the development server

On the landing page click on the forget password on the sign-in page menu

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -7,7 +7,13 @@ module.exports = {
         josefin: ["Josefin Slab", "serif"],
       },
     },
+    screen: {
+
+      'sm':'425px',
+      'md':'768px',
+      'xl':'1024px',
```
Comment (PrinceNiyonshuti) [inline tailwind.config.js]: why you used sm,md,xl,2xl  , cant we use lg too instead of having 2xl

**Review by PrinceNiyonshuti — CHANGES_REQUESTED** [review]
please provide a detailed PR description and review the changes requested

File: tailwind.config.js (hunk 1 above)
Comment (tysepa) [reply tailwind.config.js]: but this is work well

**Review by tysepa — COMMENTED** [review]
all done

**Review by thepatrickniyo — COMMENTED** [review]
Rename your Pull request for it to be self-explanatory. 

eg: **feat reset password #31**

**Review by tysepa — COMMENTED** [review]
I have made the change on this PR

**Review by PrinceNiyonshuti — APPROVED** [review]
looks good

**Review by Niyonkuru1 — APPROVED** [review]

**Review by rachelakimana — APPROVED** [review]
Looks good

**Pull request merged**